nautobot/docs/development/core/getting-started.md

```diff
@@ -266,7 +266,7 @@ You may install Poetry in your user environment by running:
 curl -sSL https://install.python-poetry.org | python3 -
 ```
 
-+/- 1.5.6
+!!! tip
     Poetry 1.3+ is required to be able to install packages against the latest lockfile.
```

Comment:
I'd remove this - we're currently using poetry 1.8 for the lockfile and I don't think this note provides much value.